portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,6 +48,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String json = convertToJsonUsingGson(quotes);
     response.setContentType("application/json;");
     response.getWriter().println(json);
+
```

Comment:
No need to create variables for the quotes, you can simply write
quotes.add("I am in a charming state of confusion. - Ada Lovelace"); 
etc.